File: Bank/src/main/Exit.java
package main;

import java.util.Scanner;

public class Exit {
    public static boolean close(){
        boolean resBo = false;
        boolean flag = true;
        Scanner scan = new Scanner(System.in);
        String res;
        while (flag) {
            System.out.println("Desea realizar otra operacion(s/n)?");
            res = scan.next();
            if (res.equals("s") || res.equals("S")) {
                flag = false;
                resBo = true;
            } else if (res.equals("n") || res.equals("N")) {
                flag = false;
                resBo = false;
                System.out.println("Usted salio");
            }
        }
        return resBo;
    }
}

File: BankPy/Exit.py
def close():
    resbo = False
    flag = True

    while flag:
        res = input("Desea realizar otra operacion(s/n)?")
        if res == "s" or res == "S":
            flag = False
            resbo = True
        elif res == "n" or res == "N":
            flag = False
            resbo = False
            print("Usted Salio")
    return resbo
